app/boards/shields/breeze/breeze.dtsi

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <dt-bindings/zmk/matrix_transform.h>
+
+/ {
+	chosen {
+		zmk,kscan = &kscan0;
+		zmk,matrix_transform = &default_transform;
+	};
+
+	default_transform: keymap_transform_0 {
```

Comment:
It would be nice to add a transform for handling folks who snap off the arrow cluster/macro portion on the right side as well, and just use it as a mirrored split.

Reply:
@petejohanson OK I think I did the right thing in [83d3c08](https://github.com/zmkfirmware/zmk/pull/1125/commits/83d3c08fa4268d90b4f79259d6b6bb2b43c7301a)?